src/Sulu/Bundle/RouteBundle/Tests/Functional/Entity/RouteRepositoryTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\RouteBundle\Tests\Functional\Controller;
+
+use Sulu\Bundle\RouteBundle\Entity\Route;
+use Sulu\Bundle\RouteBundle\Entity\RouteRepositoryInterface;
+use Sulu\Bundle\TestBundle\Testing\SuluTestCase;
+
+class RouteRepositoryTest extends SuluTestCase
+{
+    public function testRemove(): void
+    {
+        $this->purgeDatabase();
```

Comment:
Don't you want to move that to the `setUp` method already? That will be necessary as soon as the second test is implemented.